piksi_tools/bootload.py

```diff
@@ -88,6 +105,10 @@ def wait_for_handshake(self, timeout=None):
     return True
 
   def reply_handshake(self):
+    """
+    Send a handshake to Piksi bootloader to request bootloader mode. Should be
+    sent after receiving handshake from Piksi bootloader.
+    """
     # < 0.45 of SBP protocol, reuse single handshake message.
     if self.sbp_version < (0, 45):
       self.link.send(SBP_MSG_BOOTLOADER_HANDSHAKE_DEPRECATED, '\x00')
```

Comment:
You should be able to construct an object inline and send that over. 

Construction example for another message:
https://github.com/swift-nav/sbp_log_analysis/blob/master/tests/test_rtk_fix_loss.py#L29

From, https://github.com/swift-nav/libsbp/blob/master/python/sbp/deprecated.py#L28,
```
MsgBootloaderHandshakeDeprecated(0x1337, 0).pack 
```
should work.